docs/runtime_suite/auth0-client/50_usage.md

```diff
@@ -173,16 +177,52 @@ You should instead implement the signup logic in another service, that takes car
 
 :::
 
-This API can be contacted with the same body accepted by Auth0 [POST /api/v2/users](https://auth0.com/docs/api/management/v2/users/post-users) API, with the only difference that the `connection` parameter is optional: if omitted will default on the `managementClient.defaultCreateUserConnection` configuration parameter.
+This API can be contacted with the same body accepted by Auth0 [POST /api/v2/users](https://auth0.com/docs/api/management/v2/users/post-users) API, with the only difference that the `connection` parameter is optional; when omitted it will default on the configured:
+- `managementClient.defaultManagementConnectionName` configuration parameter
+- `managementClient.defaultCreateUserConnection` configuration parameter (deprecated)
+
+If none of the above is defined, the api will throw an error.
+
+Refer to the [configuration schema](./20_configuration.md#configuration)
 
 ### Patch a user
 
 The endpoint `PATCH /users/:userID` allows to patch a user.
 
 This API can be contacted with the same body accepted by Auth0 [PATCH /api/v2/users/{id}](https://auth0.com/docs/api/management/v2/users/patch-users-by-id) API.
 
-### Delete a user
+#### Delete a user
 
 The endpoint `DELETE /users/:userID` allows to delete a user.
 
 It takes no additional parameters.
+
+### Auth0 Jobs endpoints
+
+Auth0 Jobs endpoints are designed for bulk operation on the underlying tenant.
```

Comment:
```suggestion
Auth0 Jobs endpoints are designed for bulk operation on the underlying tenant.

:::info
These endpoints are available from `v3.4.0`
:::
```

Reply:
@malta895 can you add this to the original repo as well please?

Reply:
yep, will do in a PR I'm working on right now!